File: bmi.cpp
/*
 * bmi.cpp
 * 
 *
 */


#include <iostream>
using namespace std;


int main(int argc, char **argv)
{
    float wzrost = 0;
    int waga = 0;
    float bmi = 0;
    cout<<"podaj wzrost: ";
    cin>> wzrost;
    cout<<"podaj swoja wage: ";
    cin>>waga;
    
    bmi = waga/(wzrost*wzrost);
    
    if(bmi>=30)
    {
        cout<<"otyłość";
    }    
    else if (bmi >= 25)
    {
    cout<<"nadwaga";    
    }   
    else if (bmi >= 18.5)
    {
        cout<<"norma";
    }
    else
    {
        cout<<"niedowaga";
    }
    cout<<bmi;        
	return 0;
}

File: bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    wzrost = float(input("podaj wzrost: "))
    waga = int(input("podaj mase: "))
    bmi = waga / (wzrost * wzrost)
    print(bmi)

    while bmi>=0:

        if bmi >= 30:
            print("otyłość")
        elif bmi >= 25:
            print("niedowaga")
        elif bmi >= 18.5:
            print("norma")
        else:
            print("niedowaga")
    wzrost = float(input("podaj wzrost: "))
    waga = int(input("podaj mase: "))
    bmi = waga / (wzrost * wzrost)
    print(bmi)
    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
